### PR TITLE
Adjust rocket interaction test

### PR DESCRIPTION
**Pull request by MattGomb**
In this PR I finish the testing files.

### CODE REVIEW

File: src/components/Rockets.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const Rocket = (props) => {
         <div>
           <span className={style.rocketName}>{name}</span>
         </div>
-        <div className={style.description} data-testid="reserved">
+        <div className={style.description}>
           {reserved && (
             <span className={style.badge}>Reserved</span>
           )}
```
Comment (uwadonat) [inline src/components/Rockets.js:31]: - [x] Kindly add some styles to have space at the left side of these rocket images.

image

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @Team, :hand: 

Good job so far! :mechanical_arm: 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>
### To highlight! :+1: 
- Well organized project ✔️
- Test of the project pass successfully ✔️
- No linter errors ✔️
- The application works as expected ✔️

## Status:Required Changes ♻️
- [x] Kindly customize your readme file to the project. Remove unneeded lines,  uncomment needed link and commands like instructions to run/test the project. 
- [x] _Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Rockets.js (hunk 1 above)
Comment (MattGomb) [reply src/components/Rockets.js:31]: Will do, thanks!

**Review by levy002 — APPROVED** [review]
## Hello @MattGomb & @Goruchie 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly make sure to add more descriptive PR in order to make your repo more professional. :wink: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**